src-files/def-config.yaml

```diff
@@ -4,14 +4,14 @@
 
 # Parameters of repository traversal.
 traversal:
-  # Folders which we pretend do not exist
+  # Glob patterns describing files which we pretend do not exist
   # (so they are neither analyzed nor can be referenced).
   ignored:
     # Git files
-    - .git
+    - .git/**/*
```

Comment:
Is it equivalent to `.git/`?

Reply:
In glob language (at least supposed by library I use), this means "match any file in `.git` folder", and this is the only way to express that.

Reply:
I see there is `filterExcludedDirs` in `gatherRepoInfo`, how is it supposed to work? Which dirs are excluded?

Reply:
I just came to the same thing :man_facepalming: 

Apparently, here I should still expect folder (or, alternatively, file) name, this should be easy to handle without globs.